### ISSUE TITLE
Run code analyzer on test projects to search for missing Fact/Theory attributes

### ISSUE THREAD

**Issue by ViktorHofer** [dotnet/corefx#38022]
See https://github.com/dotnet/corefx/pull/37451#discussion_r288595192

@danmosemsft could be a good task for an intern / community.

cc @stephentoub

**Comment by stephentoub** [comment]
cc: @safern

**Comment by safern** [comment]
I started this work but then my priorities changed. I think it should be fairly simple, we can enable the analyzers like I did here: https://github.com/dotnet/corefx/pull/34410 and then disable all warnings except the ones for missing Fact/Theory attributes.

I'll leave it unassigned in case someone from the community wants to pick it up (since it is not marked as 3.0) -- if after ZBB is still unresolved I will finish it.
